sample-project/App.fsproj

```diff
@@ -11,7 +11,8 @@
         <PackageReference Include="Fable.Browser.Dom" Version="2.14.0" />
         <PackageReference Include="Fable.Core" Version="4.3.0" />
         <PackageReference Include="Nojaf.Fable.React" Version="0.0.2" />
-        <PackageReference Include="Thoth.Json" Version="10.2.0" />
+        <PackageReference Include="Thoth.Json" Version="10.4.1" />
```

Comment:
These changes are weird, I would not expect the sample to change.

Reply:
it's a minor though so should not be breaking on apis